### PR TITLE
feat(db/locales): fix missing zhTW CompletedText locale

### PR DESCRIPTION
**Pull request by Bogir**
Texts for world event quest.
Taken from TrinityCore/issues/27865

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1646922359631359324.sql [hunk 1]
```diff
@@ -0,0 +1,447 @@
...
...1835,11836,11837,11838,11839,11840,11841,11842,11843,11844,11845,11846,11847,11848,11849,11850,11851,11852,11853,11854,11855,11856,11857,11858,11859,11860,11937,12361,12362,12363,12364,12365,12366,12367,12368,12369,12370,12371,12373,12374,12375,12376,12377,12378,12379,12380,12381,12382,12383,12384,12385,12386,12387,12388,12389,12390,12391,12392,12393,12394,12395,12396,12397,12398,12399,12400,12401,12402,12403,12404,12405,12406,12407,12408,12409,12410,12940,12941,12946,12947,12950,13065,13440,13441,13442,13443,13444,13445,13446,13447,13449,13450,13451,13452,13453,13454,13455,13456,13457,13458,13459,13460,13461,13462,13463,13464,13465,13466,13467,13468,13469,13470,13471,13472,13474,13493,13494,13495,13496,13497,13498,13499,13500,13501,13548,13952,13966,14022,14023,14024,14028,14030,14033,14035,14036,14037,14040,14041,14043,14044,14047,14048,14051,14053,14054,14055,14058,14059,14060,14061,14062,14064,14065,14166,14171,14172,14173,14174,14175,14176,14177,14483,14488,24536,24541,24576,24597,24609,24610,24611,24612,24613,24614,24615,24629,24635,24636,24645,24647,24650,24651,24652,24655,24656,24657,24659,24660,24662,24664,24665,24666,24745,24792,24793,24804,24805,24848,24849,24850,24851);
```
Comment (LukasVolgger) [inline data/sql/updates/pending_db_world/rev_1646922359631359324.sql:3]: Please add backticks around table and column names. Keywords in uppercase please

**Review by LukasVolgger — CHANGES_REQUESTED** [review]

**Review by LukasVolgger — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1646922359631359324.sql [hunk 2]
```diff
@@ -0,0 +1,447 @@
...
+INSERT INTO `quest_template_locale` VALUES
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1646922359631359324.sql:4]: Add table columns, please.

**Review by Nefertumm — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1646922359631359324.sql (hunk 2 above)
Comment (LukasVolgger) [reply data/sql/updates/pending_db_world/rev_1646922359631359324.sql:4]: You're absolutely right, i missed that! Sry!

Comment (Bogir) [reply data/sql/updates/pending_db_world/rev_1646922359631359324.sql]: Added

File: data/sql/updates/pending_db_world/rev_1646922359631359324.sql [hunk 3]
```diff
@@ -0,0 +1,447 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1646922359631359324');
+
+REPLACE INTO `quest_template_locale` (ID, locale, Title, Details, Objectives, EndText, CompletedText, ObjectiveText1, ObjectiveText2, ObjectiveText3,  ObjectiveText4, VerifiedBuild)
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1646922359631359324.sql:3]: backticks too please :D

**Comment by ZhengPeiRu21** [comment]
This texts look like correct zhTW texts. Thanks Bogir!

**Pull request merged**